pkg/csi/service/common/authmanager.go

```diff
@@ -384,6 +389,14 @@ func getDsToFileServiceEnabledMap(ctx context.Context, vc *vsphere.VirtualCenter
 			log.Errorf("failed to get the vsan cluster config. error: %+v", err)
 			return nil, err
 		}
+		if !(*config.Enabled) {
```

Comment:
Can we handle nil config?

Reply:
Config can never be nil if err is empty. 
Config will be nil only if err is non-empty.